modules/Head/Branding.tsx

```diff
@@ -9,7 +9,7 @@ const DynamicPreviewBranding = dynamic(
         const component = await import('./components/DynamicPreviewBranding');
         return { default: component.DynamicPreviewBranding };
     },
-    { ssr: false },
+    { ssr: true },
```

Comment:
`DynamicPreviewBranding` is a client component, are you sure we can render it on the server?

Reply:
Yeah I tested to make sure it works, Next 15 does not allow importing with `ssr: false` in a server component